Emperor's Children Fixes

Medonoop opened the pull request:
## Added Units

Kakophoni Squad
Sun Killer Squad
Rylanor the Unyielding

## Fixed Units

Fulgirm
Captain Saul Tarvitz
Captain Lucius
Palatine Blade Squad
Phoenix Terminator Squad
Vulkan

## Not done yet

Phoenix Warden Consul

## Test Case

![image](https://user-images.githubusercontent.com/25377393/183251623-466da00b-3028-4ae4-bd1d-6bb66d79f132.png)
![image](https://user-images.githubusercontent.com/25377393/183251647-76038c2f-0224-4e21-ae8a-e2062827d729.png)
![image](https://user-images.githubusercontent.com/25377393/183251674-905a61bf-edd4-4ac7-abb9-d24fa7a7f58c.png)
![image](https://user-images.githubusercontent.com/25377393/183251706-12ef3b1f-814a-4cbc-bfa5-53cbcf52e1cc.png)
![image](https://user-images.githubusercontent.com/25377393/183251692-5da7690c-d657-47bd-abd0-55ca00bb773c.png)

Medonoop commented on 2022 - Horus Heresy.gst:
```diff
@@ -123,9 +123,14 @@
+        <infoLink id="2a4b-7961-1ac1-06d9" name="Bulky (X)" hidden="false" targetId="676c-7b75-4b6f-9405" type="rule">
           <modifiers>
             <modifier type="set" field="name" value="Bulky (4)"/>
+            <modifier type="set" field="hidden" value="true">
```
Added condition to hide this when using EC so Fulgrim can show Bulky (6) by itself

nstephenh replied:
I believe we tend to use "set name to bulky (6)" for this type of mod.

Medonoop replied:
That is on Fulgrim's profile. This is to hide the original Bulky (4) which is printed off for all Primarchs. Otherwise you get a Bulky (4) and Bulky (6) on Fulgrim's rules.

UrsinePatriarch commented on 2022 - Legiones Astartes.cat:
```diff
@@ -36114,6 +36174,223 @@ A Perfect Fury: If a model or models with the Skill Unmatched special rule selec
+            <constraint field="selections" scope="parent" value="1.0" percentValue="false" shared="true" includeChildSelections="false" includeChildForces="false" id="07bc-438e-ddb3-c8fa" type="min"/>
+          </constraints>
+          <profiles>
+            <profile id="c8cf-9df4-1a7f-1522" name="Pheonix Champion" hidden="false" typeId="4bb2-cb95-e6c8-5a21" typeName="Unit">
```
"Pheonix"

Medonoop replied:
Fixed typo

UrsinePatriarch reviewed (changes requested):
Seems like you have 6 options selected in Palatine Blades but only 5 models; are you counting the Dedicated Transport by accident?

If the Navaetor is exchanging their chainsword/combat blade, shouldn't its profile be in there, as well? Essentially the default option being the chainsword and the others being selections?

Loyalist/Traitor isn't actually needed as a rule since it's mostly for categorizing things which we're already doing in other ways.
It's just "Set Hidden = false if selection in force =1 of X/Y"

Medonoop commented:
Fixed Palatine Blades counting the transport for weapon counts. 

Moved Novaetor weapon options onto it. The Novaetor can swap the chainsword or the combat blade for a melee weapon, but no one in their right mind would swap the chainsword instead of a combat blade so the chainsword should always show for them.

Removed any Traitor/Loyalist rules from EC units.

Medonoop commented:
Added Retinue options to Fulgrim.

Fixed max squad size for Phoenix Terminators when they take a standard bearer.
